ext/RMagick/extconf.rb

```diff
@@ -87,27 +89,35 @@ def configure_compile_options
 
       elsif RUBY_PLATFORM =~ /mingw/ # mingw
 
-        `identify -version` =~ /Version: ImageMagick (\d+\.\d+\.\d+)-+\d+ /
+        if find_executable 'magick'
```

Comment:
Thanks for your initial comments. These changes are in the last two commits of the PR. It will be easier for me to make these changes once the rest has been merged.

Reply:
Just made some changes for this.